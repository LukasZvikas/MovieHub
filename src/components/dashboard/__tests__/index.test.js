import { mount } from "@vue/test-utils";
import CarouselWrapper from "../";
jest.mock("../../../utilities/fetch");

const search_data = [
  {
    adult: false,
    backdrop_path: "/wqtaHWOEZ3rXDJ8c6ZZShulbo18.jpg",
    id: 332562,
    overview:
      "Seasoned musician Jackson Maine discovers—and falls in love with—struggling artist Ally. She has just about given up on her dream to make it big as a singer—until Jack coaxes her into the spotlight. But even as Ally's career takes off, the personal side of their relationship is breaking down, as Jack fights an ongoing battle with his own internal demons.",
    poster_path: "/wrFpXMNBRj2PBiN4Z5kix51XaIZ.jpg",
    release_date: "2018-10-03",
    title: "A Star Is Born",
    vote_average: 7.5
  }
];

describe("CarouselWrapper", () => {
  let wrapper;
  beforeEach((done) => {
    wrapper = mount(CarouselWrapper);
    wrapper.vm.$nextTick(() => {
      wrapper.setData({
        popularMovies: [...search_data],
        comingSoon: [...search_data]
      });
      done();
    });
  });

  it("renders two carousels", () => {
    expect(wrapper.findAll('[data-test="carousel"]')).toHaveLength(2);
  });
  it("renders two movie carousel item", () => {
    expect(wrapper.findAll('[data-test="carousel-item"]')).toHaveLength(2);
  });

  describe("Popular Movies", () => {
    it("carousel has a correct heading", () => {
      expect(wrapper.vm.$el).toMatchSnapshot();
      expect(wrapper.find('[data-test="carousel-1"]').text()).toBe(
        "Popular Movies"
      );
    });

    it("movie carousel item img has a correct src", () => {
      expect(
        wrapper.find('[data-test="carousel-item-img"]').attributes("src")
      ).toBe("http://image.tmdb.org/t/p/w500/wrFpXMNBRj2PBiN4Z5kix51XaIZ.jpg");
    });
    it("movie carousel item has a correct movie title", () => {
      expect(wrapper.find('[data-test="carousel-item-title"]').text()).toBe(
        "A Star Is Born"
      );
    });
  });
  describe("Coming Soon Movies", () => {
    it("carousel has a correct heading", () => {
      expect(wrapper.find('[data-test="carousel-2"]').text()).toBe(
        "Coming Soon"
      );
    });

    it("movie carousel item img has a correct src", () => {
      expect(
        wrapper.find('[data-test="carousel-item-img"]').attributes("src")
      ).toBe("http://image.tmdb.org/t/p/w500/wrFpXMNBRj2PBiN4Z5kix51XaIZ.jpg");
    });
    it("movie carousel item has a correct movie title", () => {
      expect(wrapper.find('[data-test="carousel-item-title"]').text()).toBe(
        "A Star Is Born"
      );
    });
  });
});
