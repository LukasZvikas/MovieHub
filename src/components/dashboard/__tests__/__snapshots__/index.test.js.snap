// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CarouselWrapper Coming Soon Movies carousel has a correct heading 1`] = `
<div
  class="dashboard-wrap"
>
  <div
    class="movie-list-wrapper"
  >
    <div
      style="color: rgb(255, 255, 255); display: flex; justify-content: center; font-size: 2.2rem;"
    >
      <div
        class="movie-list-title"
        data-test="carousel-1"
      >
        Popular Movies
      </div>
    </div>
     
    <div
      class="carousel slide"
      data-ride="carousel"
      id="carousel-1"
    >
      <div
        class="carousel-inner"
      >
        <div
          class="carousel-item active"
          data-test="carousel"
        >
          <div
            class="row"
            style="padding: 0px;"
          >
            <div>
              <div
                class="movie-item-box"
                data-test="carousel-item"
              >
                <img
                  class="img-fluid"
                  data-test="carousel-item-img"
                  src="http://image.tmdb.org/t/p/w500/wrFpXMNBRj2PBiN4Z5kix51XaIZ.jpg"
                  style="height: 100%;"
                />
                 
                <div
                  class="movie-item-box-title"
                  data-test="carousel-item-title"
                >
                  A Star Is Born
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
       
      <a
        class="carousel-control-prev"
        data-slide="prev"
        href="#carousel-1"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-prev-icon"
        />
         
        <span
          class="sr-only"
        >
          Previous
        </span>
      </a>
       
      <a
        class="carousel-control-next"
        data-slide="next"
        href="#carousel-1"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-next-icon"
        />
         
        <span
          class="sr-only"
        >
          Next
        </span>
      </a>
    </div>
  </div>
   
  <div
    class="movie-list-wrapper"
  >
    <div
      style="color: rgb(255, 255, 255); display: flex; justify-content: center; font-size: 2.2rem;"
    >
      <div
        class="movie-list-title"
        data-test="carousel-2"
      >
        Coming Soon
      </div>
    </div>
     
    <div
      class="carousel slide"
      data-ride="carousel"
      id="carousel-2"
    >
      <div
        class="carousel-inner"
      >
        <div
          class="carousel-item active"
          data-test="carousel"
        >
          <div
            class="row"
            style="padding: 0px;"
          >
            <div>
              <div
                class="movie-item-box"
                data-test="carousel-item"
              >
                <img
                  class="img-fluid"
                  data-test="carousel-item-img"
                  src="http://image.tmdb.org/t/p/w500/wrFpXMNBRj2PBiN4Z5kix51XaIZ.jpg"
                  style="height: 100%;"
                />
                 
                <div
                  class="movie-item-box-title"
                  data-test="carousel-item-title"
                >
                  A Star Is Born
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
       
      <a
        class="carousel-control-prev"
        data-slide="prev"
        href="#carousel-2"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-prev-icon"
        />
         
        <span
          class="sr-only"
        >
          Previous
        </span>
      </a>
       
      <a
        class="carousel-control-next"
        data-slide="next"
        href="#carousel-2"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-next-icon"
        />
         
        <span
          class="sr-only"
        >
          Next
        </span>
      </a>
    </div>
  </div>
</div>
`;

exports[`CarouselWrapper Coming Soon Movies movie carousel item has a correct movie title 1`] = `
<div
  class="dashboard-wrap"
>
  <div
    class="movie-list-wrapper"
  >
    <div
      style="color: rgb(255, 255, 255); display: flex; justify-content: center; font-size: 2.2rem;"
    >
      <div
        class="movie-list-title"
        data-test="carousel-1"
      >
        Popular Movies
      </div>
    </div>
     
    <div
      class="carousel slide"
      data-ride="carousel"
      id="carousel-1"
    >
      <div
        class="carousel-inner"
      >
        <div
          class="carousel-item active"
          data-test="carousel"
        >
          <div
            class="row"
            style="padding: 0px;"
          >
            <div>
              <div
                class="movie-item-box"
                data-test="carousel-item"
              >
                <img
                  class="img-fluid"
                  data-test="carousel-item-img"
                  src="http://image.tmdb.org/t/p/w500/wrFpXMNBRj2PBiN4Z5kix51XaIZ.jpg"
                  style="height: 100%;"
                />
                 
                <div
                  class="movie-item-box-title"
                  data-test="carousel-item-title"
                >
                  A Star Is Born
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
       
      <a
        class="carousel-control-prev"
        data-slide="prev"
        href="#carousel-1"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-prev-icon"
        />
         
        <span
          class="sr-only"
        >
          Previous
        </span>
      </a>
       
      <a
        class="carousel-control-next"
        data-slide="next"
        href="#carousel-1"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-next-icon"
        />
         
        <span
          class="sr-only"
        >
          Next
        </span>
      </a>
    </div>
  </div>
   
  <div
    class="movie-list-wrapper"
  >
    <div
      style="color: rgb(255, 255, 255); display: flex; justify-content: center; font-size: 2.2rem;"
    >
      <div
        class="movie-list-title"
        data-test="carousel-2"
      >
        Coming Soon
      </div>
    </div>
     
    <div
      class="carousel slide"
      data-ride="carousel"
      id="carousel-2"
    >
      <div
        class="carousel-inner"
      >
        <div
          class="carousel-item active"
          data-test="carousel"
        >
          <div
            class="row"
            style="padding: 0px;"
          >
            <div>
              <div
                class="movie-item-box"
                data-test="carousel-item"
              >
                <img
                  class="img-fluid"
                  data-test="carousel-item-img"
                  src="http://image.tmdb.org/t/p/w500/wrFpXMNBRj2PBiN4Z5kix51XaIZ.jpg"
                  style="height: 100%;"
                />
                 
                <div
                  class="movie-item-box-title"
                  data-test="carousel-item-title"
                >
                  A Star Is Born
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
       
      <a
        class="carousel-control-prev"
        data-slide="prev"
        href="#carousel-2"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-prev-icon"
        />
         
        <span
          class="sr-only"
        >
          Previous
        </span>
      </a>
       
      <a
        class="carousel-control-next"
        data-slide="next"
        href="#carousel-2"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-next-icon"
        />
         
        <span
          class="sr-only"
        >
          Next
        </span>
      </a>
    </div>
  </div>
</div>
`;

exports[`CarouselWrapper Coming Soon Movies movie carousel item img has a correct src 1`] = `
<div
  class="dashboard-wrap"
>
  <div
    class="movie-list-wrapper"
  >
    <div
      style="color: rgb(255, 255, 255); display: flex; justify-content: center; font-size: 2.2rem;"
    >
      <div
        class="movie-list-title"
        data-test="carousel-1"
      >
        Popular Movies
      </div>
    </div>
     
    <div
      class="carousel slide"
      data-ride="carousel"
      id="carousel-1"
    >
      <div
        class="carousel-inner"
      >
        <div
          class="carousel-item active"
          data-test="carousel"
        >
          <div
            class="row"
            style="padding: 0px;"
          >
            <div>
              <div
                class="movie-item-box"
                data-test="carousel-item"
              >
                <img
                  class="img-fluid"
                  data-test="carousel-item-img"
                  src="http://image.tmdb.org/t/p/w500/wrFpXMNBRj2PBiN4Z5kix51XaIZ.jpg"
                  style="height: 100%;"
                />
                 
                <div
                  class="movie-item-box-title"
                  data-test="carousel-item-title"
                >
                  A Star Is Born
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
       
      <a
        class="carousel-control-prev"
        data-slide="prev"
        href="#carousel-1"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-prev-icon"
        />
         
        <span
          class="sr-only"
        >
          Previous
        </span>
      </a>
       
      <a
        class="carousel-control-next"
        data-slide="next"
        href="#carousel-1"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-next-icon"
        />
         
        <span
          class="sr-only"
        >
          Next
        </span>
      </a>
    </div>
  </div>
   
  <div
    class="movie-list-wrapper"
  >
    <div
      style="color: rgb(255, 255, 255); display: flex; justify-content: center; font-size: 2.2rem;"
    >
      <div
        class="movie-list-title"
        data-test="carousel-2"
      >
        Coming Soon
      </div>
    </div>
     
    <div
      class="carousel slide"
      data-ride="carousel"
      id="carousel-2"
    >
      <div
        class="carousel-inner"
      >
        <div
          class="carousel-item active"
          data-test="carousel"
        >
          <div
            class="row"
            style="padding: 0px;"
          >
            <div>
              <div
                class="movie-item-box"
                data-test="carousel-item"
              >
                <img
                  class="img-fluid"
                  data-test="carousel-item-img"
                  src="http://image.tmdb.org/t/p/w500/wrFpXMNBRj2PBiN4Z5kix51XaIZ.jpg"
                  style="height: 100%;"
                />
                 
                <div
                  class="movie-item-box-title"
                  data-test="carousel-item-title"
                >
                  A Star Is Born
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
       
      <a
        class="carousel-control-prev"
        data-slide="prev"
        href="#carousel-2"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-prev-icon"
        />
         
        <span
          class="sr-only"
        >
          Previous
        </span>
      </a>
       
      <a
        class="carousel-control-next"
        data-slide="next"
        href="#carousel-2"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-next-icon"
        />
         
        <span
          class="sr-only"
        >
          Next
        </span>
      </a>
    </div>
  </div>
</div>
`;

exports[`CarouselWrapper Popular Movies carousel has a correct heading 1`] = `
<div
  class="dashboard-wrap"
>
  <div
    class="movie-list-wrapper"
  >
    <div
      style="color: rgb(255, 255, 255); display: flex; justify-content: center; font-size: 2.2rem;"
    >
      <div
        class="movie-list-title"
        data-test="carousel-1"
      >
        Popular Movies
      </div>
    </div>
     
    <div
      class="carousel slide"
      data-ride="carousel"
      id="carousel-1"
    >
      <div
        class="carousel-inner"
      >
        <div
          class="carousel-item active"
          data-test="carousel"
        >
          <div
            class="row"
            style="padding: 0px;"
          >
            <div>
              <div
                class="movie-item-box"
                data-test="carousel-item"
              >
                <img
                  class="img-fluid"
                  data-test="carousel-item-img"
                  src="http://image.tmdb.org/t/p/w500/wrFpXMNBRj2PBiN4Z5kix51XaIZ.jpg"
                  style="height: 100%;"
                />
                 
                <div
                  class="movie-item-box-title"
                  data-test="carousel-item-title"
                >
                  A Star Is Born
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
       
      <a
        class="carousel-control-prev"
        data-slide="prev"
        href="#carousel-1"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-prev-icon"
        />
         
        <span
          class="sr-only"
        >
          Previous
        </span>
      </a>
       
      <a
        class="carousel-control-next"
        data-slide="next"
        href="#carousel-1"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-next-icon"
        />
         
        <span
          class="sr-only"
        >
          Next
        </span>
      </a>
    </div>
  </div>
   
  <div
    class="movie-list-wrapper"
  >
    <div
      style="color: rgb(255, 255, 255); display: flex; justify-content: center; font-size: 2.2rem;"
    >
      <div
        class="movie-list-title"
        data-test="carousel-2"
      >
        Coming Soon
      </div>
    </div>
     
    <div
      class="carousel slide"
      data-ride="carousel"
      id="carousel-2"
    >
      <div
        class="carousel-inner"
      >
        <div
          class="carousel-item active"
          data-test="carousel"
        >
          <div
            class="row"
            style="padding: 0px;"
          >
            <div>
              <div
                class="movie-item-box"
                data-test="carousel-item"
              >
                <img
                  class="img-fluid"
                  data-test="carousel-item-img"
                  src="http://image.tmdb.org/t/p/w500/wrFpXMNBRj2PBiN4Z5kix51XaIZ.jpg"
                  style="height: 100%;"
                />
                 
                <div
                  class="movie-item-box-title"
                  data-test="carousel-item-title"
                >
                  A Star Is Born
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
       
      <a
        class="carousel-control-prev"
        data-slide="prev"
        href="#carousel-2"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-prev-icon"
        />
         
        <span
          class="sr-only"
        >
          Previous
        </span>
      </a>
       
      <a
        class="carousel-control-next"
        data-slide="next"
        href="#carousel-2"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-next-icon"
        />
         
        <span
          class="sr-only"
        >
          Next
        </span>
      </a>
    </div>
  </div>
</div>
`;

exports[`CarouselWrapper Popular Movies movie carousel item has a correct movie title 1`] = `
<div
  class="dashboard-wrap"
>
  <div
    class="movie-list-wrapper"
  >
    <div
      style="color: rgb(255, 255, 255); display: flex; justify-content: center; font-size: 2.2rem;"
    >
      <div
        class="movie-list-title"
        data-test="carousel-1"
      >
        Popular Movies
      </div>
    </div>
     
    <div
      class="carousel slide"
      data-ride="carousel"
      id="carousel-1"
    >
      <div
        class="carousel-inner"
      >
        <div
          class="carousel-item active"
          data-test="carousel"
        >
          <div
            class="row"
            style="padding: 0px;"
          >
            <div>
              <div
                class="movie-item-box"
                data-test="carousel-item"
              >
                <img
                  class="img-fluid"
                  data-test="carousel-item-img"
                  src="http://image.tmdb.org/t/p/w500/wrFpXMNBRj2PBiN4Z5kix51XaIZ.jpg"
                  style="height: 100%;"
                />
                 
                <div
                  class="movie-item-box-title"
                  data-test="carousel-item-title"
                >
                  A Star Is Born
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
       
      <a
        class="carousel-control-prev"
        data-slide="prev"
        href="#carousel-1"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-prev-icon"
        />
         
        <span
          class="sr-only"
        >
          Previous
        </span>
      </a>
       
      <a
        class="carousel-control-next"
        data-slide="next"
        href="#carousel-1"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-next-icon"
        />
         
        <span
          class="sr-only"
        >
          Next
        </span>
      </a>
    </div>
  </div>
   
  <div
    class="movie-list-wrapper"
  >
    <div
      style="color: rgb(255, 255, 255); display: flex; justify-content: center; font-size: 2.2rem;"
    >
      <div
        class="movie-list-title"
        data-test="carousel-2"
      >
        Coming Soon
      </div>
    </div>
     
    <div
      class="carousel slide"
      data-ride="carousel"
      id="carousel-2"
    >
      <div
        class="carousel-inner"
      >
        <div
          class="carousel-item active"
          data-test="carousel"
        >
          <div
            class="row"
            style="padding: 0px;"
          >
            <div>
              <div
                class="movie-item-box"
                data-test="carousel-item"
              >
                <img
                  class="img-fluid"
                  data-test="carousel-item-img"
                  src="http://image.tmdb.org/t/p/w500/wrFpXMNBRj2PBiN4Z5kix51XaIZ.jpg"
                  style="height: 100%;"
                />
                 
                <div
                  class="movie-item-box-title"
                  data-test="carousel-item-title"
                >
                  A Star Is Born
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
       
      <a
        class="carousel-control-prev"
        data-slide="prev"
        href="#carousel-2"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-prev-icon"
        />
         
        <span
          class="sr-only"
        >
          Previous
        </span>
      </a>
       
      <a
        class="carousel-control-next"
        data-slide="next"
        href="#carousel-2"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-next-icon"
        />
         
        <span
          class="sr-only"
        >
          Next
        </span>
      </a>
    </div>
  </div>
</div>
`;

exports[`CarouselWrapper Popular Movies movie carousel item img has a correct src 1`] = `
<div
  class="dashboard-wrap"
>
  <div
    class="movie-list-wrapper"
  >
    <div
      style="color: rgb(255, 255, 255); display: flex; justify-content: center; font-size: 2.2rem;"
    >
      <div
        class="movie-list-title"
        data-test="carousel-1"
      >
        Popular Movies
      </div>
    </div>
     
    <div
      class="carousel slide"
      data-ride="carousel"
      id="carousel-1"
    >
      <div
        class="carousel-inner"
      >
        <div
          class="carousel-item active"
          data-test="carousel"
        >
          <div
            class="row"
            style="padding: 0px;"
          >
            <div>
              <div
                class="movie-item-box"
                data-test="carousel-item"
              >
                <img
                  class="img-fluid"
                  data-test="carousel-item-img"
                  src="http://image.tmdb.org/t/p/w500/wrFpXMNBRj2PBiN4Z5kix51XaIZ.jpg"
                  style="height: 100%;"
                />
                 
                <div
                  class="movie-item-box-title"
                  data-test="carousel-item-title"
                >
                  A Star Is Born
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
       
      <a
        class="carousel-control-prev"
        data-slide="prev"
        href="#carousel-1"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-prev-icon"
        />
         
        <span
          class="sr-only"
        >
          Previous
        </span>
      </a>
       
      <a
        class="carousel-control-next"
        data-slide="next"
        href="#carousel-1"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-next-icon"
        />
         
        <span
          class="sr-only"
        >
          Next
        </span>
      </a>
    </div>
  </div>
   
  <div
    class="movie-list-wrapper"
  >
    <div
      style="color: rgb(255, 255, 255); display: flex; justify-content: center; font-size: 2.2rem;"
    >
      <div
        class="movie-list-title"
        data-test="carousel-2"
      >
        Coming Soon
      </div>
    </div>
     
    <div
      class="carousel slide"
      data-ride="carousel"
      id="carousel-2"
    >
      <div
        class="carousel-inner"
      >
        <div
          class="carousel-item active"
          data-test="carousel"
        >
          <div
            class="row"
            style="padding: 0px;"
          >
            <div>
              <div
                class="movie-item-box"
                data-test="carousel-item"
              >
                <img
                  class="img-fluid"
                  data-test="carousel-item-img"
                  src="http://image.tmdb.org/t/p/w500/wrFpXMNBRj2PBiN4Z5kix51XaIZ.jpg"
                  style="height: 100%;"
                />
                 
                <div
                  class="movie-item-box-title"
                  data-test="carousel-item-title"
                >
                  A Star Is Born
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
       
      <a
        class="carousel-control-prev"
        data-slide="prev"
        href="#carousel-2"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-prev-icon"
        />
         
        <span
          class="sr-only"
        >
          Previous
        </span>
      </a>
       
      <a
        class="carousel-control-next"
        data-slide="next"
        href="#carousel-2"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-next-icon"
        />
         
        <span
          class="sr-only"
        >
          Next
        </span>
      </a>
    </div>
  </div>
</div>
`;

exports[`CarouselWrapper renders two carousels 1`] = `
<div
  class="dashboard-wrap"
>
  <div
    class="movie-list-wrapper"
  >
    <div
      style="color: rgb(255, 255, 255); display: flex; justify-content: center; font-size: 2.2rem;"
    >
      <div
        class="movie-list-title"
        data-test="carousel-1"
      >
        Popular Movies
      </div>
    </div>
     
    <div
      class="carousel slide"
      data-ride="carousel"
      id="carousel-1"
    >
      <div
        class="carousel-inner"
      >
        <div
          class="carousel-item active"
          data-test="carousel"
        >
          <div
            class="row"
            style="padding: 0px;"
          >
            <div>
              <div
                class="movie-item-box"
                data-test="carousel-item"
              >
                <img
                  class="img-fluid"
                  data-test="carousel-item-img"
                  src="http://image.tmdb.org/t/p/w500/wrFpXMNBRj2PBiN4Z5kix51XaIZ.jpg"
                  style="height: 100%;"
                />
                 
                <div
                  class="movie-item-box-title"
                  data-test="carousel-item-title"
                >
                  A Star Is Born
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
       
      <a
        class="carousel-control-prev"
        data-slide="prev"
        href="#carousel-1"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-prev-icon"
        />
         
        <span
          class="sr-only"
        >
          Previous
        </span>
      </a>
       
      <a
        class="carousel-control-next"
        data-slide="next"
        href="#carousel-1"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-next-icon"
        />
         
        <span
          class="sr-only"
        >
          Next
        </span>
      </a>
    </div>
  </div>
   
  <div
    class="movie-list-wrapper"
  >
    <div
      style="color: rgb(255, 255, 255); display: flex; justify-content: center; font-size: 2.2rem;"
    >
      <div
        class="movie-list-title"
        data-test="carousel-2"
      >
        Coming Soon
      </div>
    </div>
     
    <div
      class="carousel slide"
      data-ride="carousel"
      id="carousel-2"
    >
      <div
        class="carousel-inner"
      >
        <div
          class="carousel-item active"
          data-test="carousel"
        >
          <div
            class="row"
            style="padding: 0px;"
          >
            <div>
              <div
                class="movie-item-box"
                data-test="carousel-item"
              >
                <img
                  class="img-fluid"
                  data-test="carousel-item-img"
                  src="http://image.tmdb.org/t/p/w500/wrFpXMNBRj2PBiN4Z5kix51XaIZ.jpg"
                  style="height: 100%;"
                />
                 
                <div
                  class="movie-item-box-title"
                  data-test="carousel-item-title"
                >
                  A Star Is Born
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
       
      <a
        class="carousel-control-prev"
        data-slide="prev"
        href="#carousel-2"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-prev-icon"
        />
         
        <span
          class="sr-only"
        >
          Previous
        </span>
      </a>
       
      <a
        class="carousel-control-next"
        data-slide="next"
        href="#carousel-2"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-next-icon"
        />
         
        <span
          class="sr-only"
        >
          Next
        </span>
      </a>
    </div>
  </div>
</div>
`;

exports[`CarouselWrapper renders two movie carousel item 1`] = `
<div
  class="dashboard-wrap"
>
  <div
    class="movie-list-wrapper"
  >
    <div
      style="color: rgb(255, 255, 255); display: flex; justify-content: center; font-size: 2.2rem;"
    >
      <div
        class="movie-list-title"
        data-test="carousel-1"
      >
        Popular Movies
      </div>
    </div>
     
    <div
      class="carousel slide"
      data-ride="carousel"
      id="carousel-1"
    >
      <div
        class="carousel-inner"
      >
        <div
          class="carousel-item active"
          data-test="carousel"
        >
          <div
            class="row"
            style="padding: 0px;"
          >
            <div>
              <div
                class="movie-item-box"
                data-test="carousel-item"
              >
                <img
                  class="img-fluid"
                  data-test="carousel-item-img"
                  src="http://image.tmdb.org/t/p/w500/wrFpXMNBRj2PBiN4Z5kix51XaIZ.jpg"
                  style="height: 100%;"
                />
                 
                <div
                  class="movie-item-box-title"
                  data-test="carousel-item-title"
                >
                  A Star Is Born
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
       
      <a
        class="carousel-control-prev"
        data-slide="prev"
        href="#carousel-1"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-prev-icon"
        />
         
        <span
          class="sr-only"
        >
          Previous
        </span>
      </a>
       
      <a
        class="carousel-control-next"
        data-slide="next"
        href="#carousel-1"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-next-icon"
        />
         
        <span
          class="sr-only"
        >
          Next
        </span>
      </a>
    </div>
  </div>
   
  <div
    class="movie-list-wrapper"
  >
    <div
      style="color: rgb(255, 255, 255); display: flex; justify-content: center; font-size: 2.2rem;"
    >
      <div
        class="movie-list-title"
        data-test="carousel-2"
      >
        Coming Soon
      </div>
    </div>
     
    <div
      class="carousel slide"
      data-ride="carousel"
      id="carousel-2"
    >
      <div
        class="carousel-inner"
      >
        <div
          class="carousel-item active"
          data-test="carousel"
        >
          <div
            class="row"
            style="padding: 0px;"
          >
            <div>
              <div
                class="movie-item-box"
                data-test="carousel-item"
              >
                <img
                  class="img-fluid"
                  data-test="carousel-item-img"
                  src="http://image.tmdb.org/t/p/w500/wrFpXMNBRj2PBiN4Z5kix51XaIZ.jpg"
                  style="height: 100%;"
                />
                 
                <div
                  class="movie-item-box-title"
                  data-test="carousel-item-title"
                >
                  A Star Is Born
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
       
      <a
        class="carousel-control-prev"
        data-slide="prev"
        href="#carousel-2"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-prev-icon"
        />
         
        <span
          class="sr-only"
        >
          Previous
        </span>
      </a>
       
      <a
        class="carousel-control-next"
        data-slide="next"
        href="#carousel-2"
        role="button"
      >
        <span
          aria-hidden="true"
          class="carousel-control-next-icon"
        />
         
        <span
          class="sr-only"
        >
          Next
        </span>
      </a>
    </div>
  </div>
</div>
`;
